UIKITDZ/UIKITDZ/VC/CatalogBooksViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CatalogBooksViewController.swift
+//  UIKITDZ
+//
+//  Created by Алена Панченко on 05.10.2022.
+//
+
+import UIKit
+/// Экран каталога книг и покупки
+final class CatalogBooksViewController: UIViewController {
+    // Private Outlets
+    @IBOutlet weak private var bookSegmentControl: UISegmentedControl!
+    @IBOutlet weak private var bookImage: UIImageView!
```

Comment:
```suggestion
    @IBOutlet weak private var bookImageView: UIImageView!
```
если б была картинка было бы image
а так imageView